big_tests/tests/connect_SUITE.erl

```diff
@@ -546,6 +571,52 @@ bind_server_generated_resource(Config) ->
     ?assert(is_binary(Resource)),
     ?assert(byte_size(Resource) > 0).
 
+same_resource_replaces_session(Config) ->
+    UserSpec = [{resource, <<"conflict">>} | escalus_users:get_userspec(Config, alice)],
+    {ok, Alice1, _} = escalus_connection:start(UserSpec),
+
+    {ok, Alice2, _} = escalus_connection:start(UserSpec),
```

Comment:
Could you please rename 
`Alice1` -> `Alice` and `Alice2` -> `Bellatrix` 
`Alice1` -> `Geralt` and `Alice2` -> `Ciri` 
or something like that?
;)

Reply:
Not a good idea. Different names are used for different users. Different resources of the same user are numbered.